### DISCUSSION TITLE
Requesting a new Cask for FreeDV (Open Source Amateur Digital Voice)

### DISCUSSION THREAD

**Discussion by luzpaz** [Homebrew/discussions#4646] [Casks]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

Can anyone make a cask for this package please?

https://freedv.org/

Open Source Amateur Digital Voice
Where Amateur Radio Is Driving The State of the Art

Releases & Changelogs: https://github.com/drowe67/freedv-gui/releases/
v1.8.12 https://github.com/drowe67/freedv-gui/releases/download/v1.8.12/FreeDV.dmg

### Relevant casks

n/a

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Adding-Software-to-Homebrew
